### DISCUSSION TITLE
Crossfaction and Community

### DISCUSSION THREAD

**Discussion by Edoz77** [Chalsean/XFaction#359] [Ideas]
Hello !

I was a user of the addon InterGuild (https://www.curseforge.com/wow/addons/interguild) who link guild by a community. It's working almost every time, but not updated we have issues sometimes.

XFaction is very cool, but the link with Bnet friends is painfull :/. Why do not use a community? User join a community and the crossfaction is done with it, not with Bnet. Also that can include community guys like they are in the guild.

Regards,

**Comment by Chalsean** [comment]
Various non-chat use cases:

1. Last I knew community channels did not notify on achievements
2. If you are not in the same local guild, the community API will only return a very small subset of data about the other player.  Thus the roster view would be lacking multiple datasets
3. Community channel API is protected such that addons cannot send data thru it to the other player's addon, like you can with regular chat or bnet
4. Community channels have a limit of users and in very large confederates (Eternal Kingdom, Death Jesters, etc.) they would hit the cap

If your use case is strictly about merging chat, then InterGuild seems like a good solution to use.
